src/construction.cpp

```diff
@@ -1670,6 +1673,51 @@ void construct::done_digormine_stair( const tripoint_bub_ms &p, bool dig,
     tmpmap.save();
 }
 
+void construct::done_digormine_slope( const tripoint_bub_ms &p, bool dig,
+                                      Character &player_character )
+{
+    map &here = get_map();
+    const tripoint_abs_ms abs_pos = here.getglobal( p );
+    const tripoint_abs_sm pos_sm = project_to<coords::sm>( abs_pos );
+    tinymap tmpmap;
+    tmpmap.load( pos_sm + tripoint_below, false );
```

Comment:
z-levels are on, you don't need to be loading the map below us explicitly. But I see this is a copy-paste of the above function, so I request you at least make this a common static function that both these functions use instead of duplicating code.

Reply:
Alright, will get around to doing so in a bit.

Reply:
Done.